source/MockingboardCardManager.cpp

```diff
@@ -284,10 +284,10 @@ bool MockingboardCardManager::Init(void)
 		return false;
 
 	HRESULT hr = DSGetSoundBuffer(&m_mockingboardVoice, DSBCAPS_CTRLVOLUME, SOUNDBUFFER_SIZE, MockingboardCard::SAMPLE_RATE, MockingboardCard::NUM_MB_CHANNELS, "MB");
-	LogFileOutput("MBCardMgr: DSGetSoundBuffer(), hr=0x%08X\n", hr);
+	LogFileOutput("MBCardMgr: DSGetSoundBuffer(), hr=0x%08X\n", (unsigned)hr);
```

Comment:
The compiler was flagging these with a format character doesn't match data type error, so I've added explicit casts to appease it.